sdk/keyvault/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,122 @@
+
+#### Other Authentication Issues
+
+If you are using the `@azure/identity` package - which contains [DefaultAzureCredential] - to authenticate requests to Azure Key Vault, please see our [troubleshooting guide](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/Azure.Identity/Troubleshooting.md).
+
+### HTTP 403 Errors
+
+HTTP 403 errors indicate the user is not authorized to perform a specific operation in Key Vault or Managed HSM.
+
+#### Operation Not Permitted
+
+You may see an error similar to:
+
+```text
+
+RestError: Service request failed.
+Status: 403 (Forbidden)
+
+Content:
+
+{"error":{"code":"Forbidden","message":"Operation decrypt is not permitted on this key.","innererror":{"code":"KeyOperationForbidden"}}}
+
+```
+
+The operation and inner `code` may vary, but the rest of the text will indicate which operation is not permitted.
+
+This error indicates that the authenticated application or user does not have permissions to perform that operation, though the cause may vary.
+
+1. Check that the application or user has the appropriate permissions:
+    * [Access policies](https://docs.microsoft.com/azure/key-vault/general/assign-access-policy) (Key Vault)
+    * [Role-Based Access Control (RBAC)](https://docs.microsoft.com/azure/key-vault/general/rbac-guide) (Key Vault and Managed HSM)
+2. If the appropriate permissions are assigned to your application or user, make sure you are authenticating as that user.
+    * If using the [DefaultAzureCredential] a different credential might've been used than one you expected. [Enable logging](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/README.md#logging) and you will see which credential the [DefaultAzureCredential] used as shown below, and why previously-attempted credentials were rejected.
+
+## Other Service Errors
+
+To troubleshoot additional HTTP service errors not described below, see [Azure Key Vault REST API Error Codes](https://docs.microsoft.com/azure/key-vault/general/rest-error-codes).
+
+### HTTP 429: Too Many Requests
+
+If you get an exception or see logs that describe HTTP 429, you may be making too many requests to Key Vault too quickly.
+
+Possible solutions include:
+
+1. Use a singleton for any `CertificateClient`, `KeyClient`, or `SecretClient` in your application for a single Key Vault.
+2. Use a single instance of [DefaultAzureCredential] or other credential you use to authenticate your clients for each Key Vault or Managed HSM endpoint you need to access.
+3. Use Azure App Configuration for storing non-secrets and references to Key Vault secrets. Storing all app configuration in Key Vault will increase the likelihood of requests being throttled as more application instances are started. See [our @azure/app-configuration](https://github.com/Azure/azure-sdk-for-js/blob/main/appconfiguration/app-configuration/README.md) for more information.
```

Comment:
The link <https://github.com/Azure/azure-sdk-for-js/blob/main/appconfiguration/app-configuration/README.md> also doesn't exist.